From 556c2bb4c4025a5406f77a169ef671b29b0eb3fe Mon Sep 17 00:00:00 2001
From: wangjing <wangjing@codeaurora.org>
Date: Tue, 7 Apr 2015 16:22:06 +0800
Subject: [PATCH 14/20] Email: Fix the ActivityNotFoundException when click
 "Update now"

There is no activity response the intent of action VIEW and
data auth://com.android.email.ACCOUNT_SECURITY.

Add the intent filter for activity AccountSecurity, then this activity
can handle this intent.

CRs-fixed: 816922
Change-Id: I0b8982c63bd31d3fcca506fde9a1ad0d72e38cbb
---
 AndroidManifest.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 52580eee1..d455ca39a 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -346,6 +346,12 @@
             android:name=".activity.setup.AccountSecurity"
             android:label="@string/account_security_title"
             >
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:host="com.android.email.ACCOUNT_SECURITY" />
+                <data android:scheme="auth"/>
+            </intent-filter>
         </activity>
 
         <activity
-- 
2.17.1

